src/tools/miri/src/shims/windows/foreign_items.rs

```diff
@@ -427,6 +427,14 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.gen_random(ptr, len.into())?;
                 this.write_scalar(Scalar::from_bool(true), dest)?;
             }
+            "ProcessPrng" => {
+                let [ptr, len] =
+                    this.check_shim(abi, Abi::System { unwind: false }, link_name, args)?;
+                let ptr = this.read_pointer(ptr)?;
+                let len = this.read_scalar(len)?.to_target_usize(this)?;
+                this.gen_random(ptr, len.into())?;
+                this.write_scalar(Scalar::from_bool(true), dest)?;
```

Comment:
`BOOL` is an i32: https://github.com/ChrisDenton/rust/blob/fd9bb7fddedddc2ab48665214efae780dc64af2d/library/std/src/sys/pal/windows/c/windows_sys.rs#L902

Reply:
Oh, right! Context switching is fun.